### DISCUSSION TITLE
Can someone help me to get psycopg2 into the Python 3.12 layer on us-west-2?

### DISCUSSION THREAD

**Discussion by ultranol** [keithrozario/Klayers#397] [Q&A] [answered]
I click on Pull Request to try to change the file but I don't know where else to click to actually change the CSV to include Psycopg2 there. Thank you.

**Comment by keithrozario** [comment, marked as answer]
Typically you need to fork the repo, and then modify in your own fork, and then make a pull request back here. 

Hope that helps.

**Reply by ultranol** [reply]
Thank you.
